src/QirRuntime/test/Standalone/InputTypes/qir-input-types.ll

```diff
@@ -0,0 +1,49 @@
+
+%Result = type opaque
+%Range = type { i64, i64, i64 }
+%String = type opaque
+%Array = type opaque
+
+@ResultZero = external global %Result*
+@ResultOne = external global %Result*
+@PauliI = constant i2 0
+@PauliX = constant i2 1
+@PauliY = constant i2 -1
+@PauliZ = constant i2 -2
+@EmptyRange = internal constant %Range { i64 0, i64 1, i64 -1 }
+
+define void @Microsoft__Quantum__Testing__QIR__InputTypes__body(i64 %anInt, double %aDouble, i1 %aBool, %String* %aString, %Array* %anArray) {
```

Comment:
FYI: there is currently a bug in QIR generation that the array, created by the wrapper, isn't properly released (https://github.com/microsoft/qsharp-compiler/issues/844)

Reply:
Thanks for the heads up!